library/fixed_point/src/Operations.qs

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Types.FixedPoint;
+import Init.PrepareFxP;
+import Facts.IdenticalPointPosFactFxP, Facts.IdenticalFormatFactFxP, Facts.AssertAllZeroFxP;
+import Signed.Operations.Invert2sSI, Signed.Operations.MultiplySI, Signed.Operations.SquareSI;
+import Std.Arrays.Zipped;
+import Unstable.Arithmetic.RippleCarryTTKIncByLE;
+
+/// # Summary
+/// Adds a classical constant to a quantum fixed-point number.
+///
+/// # Input
+/// ## constant
+/// Constant to add to the quantum fixed-point number.
+/// ## fp
+/// Fixed-point number to which the constant will
+/// be added.
+operation AddConstantFxP(constant : Double, fp : FixedPoint) : Unit is Adj + Ctl {
```

Comment:
nitpick: It took me a moment to realize that by "Operations" you meant mathematical operations and not Q# callables that are not functions. Not sure I have a better name, just noting the terminology overloading for a namespace name.

Reply:
Hm, what about `MathOps.qs`?

Reply:
That seems reasonable to me!

Reply:
I realized I've already got an `Operations.qs` in `Signed` -- maybe we keep these the same for now, for consistency, and we can iterate the API in a follow-up PR?

Reply:
Ah, ok. Let's keep it consistent then.